CMakeLists.txt

```diff
@@ -225,7 +225,7 @@ endif (with-zbc)
 
 if (with-rbd)
 	find_library(LIBRBD rbd)
-  set(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS}, librados2, librbd1")
+	set(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS}, librados2, librbd1")
```

Comment:
Just make the first patch "Add preliminary support to build Debian packages with Cmake infrastru…" use tabs from the beginning.